### PR TITLE
Remove Liquality

### PR DESCRIPTION
**Pull request by ronaldsg20**
I've updated the `checkRloginInjectedProviders` method in order
		to ensure the Liquality injected provider will be removed from
		the web3modal providers return.

### CODE REVIEW

**Review by annipi — APPROVED** [review]
LGTM!

**Review by Freshenext — APPROVED** [review]
We spoke in persona in the hub meetings. The reason is that liquality is no longer maintained and supported. An article will be pasted here so that we know the reason for this merge.

Pending Ronald to put it.

Btw code changes are OK.

File: src/providers/injectedProviders.ts [hunk 1]
```diff
@@ -70,6 +70,10 @@ const injectedProviders = [
  * @returns updated array of providers
  */
 export const checkRLoginInjectedProviders = (providers: IProviderUserOptions[]) => {
+  // Remove liquality injected provider
+  providers.forEach((provider) => {
+    if (provider.name === 'Liquality') providers.splice(providers.indexOf(provider), 1)
+  })
```
Comment (jessgusclark) [inline src/providers/injectedProviders.ts:76]: Why are you excluding it? If the user is still using Liquality, for whatever reason, that should be their decision not the developers. Why not just allow it to be shown in the bottom with the other misc providers?

**Review by jessgusclark — CHANGES_REQUESTED** [review]
The changes to `WalletProviders.tsx` is good, but I don't understand the change to injectedProvider.

**Pull request merged**